cdap-spark-core/src/main/java/co/cask/cdap/app/runtime/spark/SparkRuntimeService.java

```diff
@@ -246,9 +246,8 @@ protected void run() throws Exception {
     } catch (Exception e) {
       // See if it is due to job cancelation. If it is, then it's not an error.
       if (jobCompletion.isCancelled()) {
-        LOG.debug("Spark program execution cancelled: {}", runtimeContext);
+        LOG.info("Spark program execution cancelled: {}", runtimeContext);
       } else {
-        LOG.error("Spark program execution failure: {}", runtimeContext, e);
```

Comment:
The exception is being throw out, the caller will log it. Logging here will cause doubling logging